index.js

```diff
@@ -0,0 +1,295 @@
+      if(display.current[0].innerHTML == 0){
+        display.current[0].innerHTML = null
+      }
+      display.current[0].innerHTML += num
+    }
+  }
+
+  var getOperand = function(symbol){
+    if(display.operand !== ''){
+      display.operand = ''
+      display.first = 0
+    }
+    switch(symbol){
+      case '+':
+        calculate()
+        display.operand = '+'
+        break
+      case '-':
+        calculate()
+        display.operand = '-'
+        break
+      case 'X':
+        calculate()
+        display.operand = '*'
+        break
+      case '*':
+        calculate()
+        display.operand = '*'
+        break
+      case '÷':
+        calculate()
+        display.operand = '/'
+        break
+      case '/':
+        calculate()
+        display.operand = '/'
+        break
+    }
+    display.firstValueAdded = true
+  }
+
+  var calculate = function() {
+    if(display.first !==0){
+      display.second = display.current[0].innerHTML
+      display.secondValueAdded = true
+      if(display.second !== 0){
+        switch(display.operand){
+          case '+':
+            display.result = parseFloat(display.first) + parseFloat(display.second)
+            break
+          case '-':
+            display.result = parseFloat(display.first) - parseFloat(display.second)
+            break
+          case '*':
+            display.result = parseFloat(display.first) * parseFloat(display.second)
+            break
+          case '/':
+            display.result = parseFloat(display.first) / parseFloat(display.second)
+            break
+        }
+        display.current[0].innerHTML = display.result
+        changeFontSize()
+      }
+    }
+  }
+
+  var allClear = function() {
+    display.current[0].innerHTML = 0
+    display.current[0].style.fontSize = "70px"
+    display.first = 0
+    display.second = 0
+    display.operand = ''
+    display.result = ''
+    display.firstValueAdded = false
+    display.secondValueAdded = false
+  }
+
+  var flipSign = function() {
+    var toFlip = display.current[0].innerHTML
+    if(toFlip > 0){
+      display.current[0].innerHTML = '-' + toFlip
+    }else{
+      var flip = display.current[0].innerHTML.indexOf('-')
+      display.current[0].innerHTML = display.current[0].innerHTML.slice(flip+1)
+    }
+  }
+
+  var percent = function() {
+    var perc = display.current[0].innerHTML
+    var removeIndex = perc.indexOf('%')
+    var toBeCalculated = perc.replace(/\% ?/g, "")
+    display.current[0].innerHTML = (toBeCalculated / 100).toFixed(2)
+  }
+
+  var keyStrokeActive = function(event) {
+    event.attributes.matches('#AC')
+
+    console.log(event.target.attributes)
+  }
+
+  var isOperand = function(sym){
+    return sym == '+' || sym == '-' || sym == '/' || sym == '*'
+  }
+  var assignClass = function(eventKey){
+    console.log(eventKey)
+    switch(eventKey){
+      case '0':
+        document.querySelector('#zero').classList
```

Comment:
You are repeating an awful lot of code here, I think you could create a helper function to make this a bit more dry.